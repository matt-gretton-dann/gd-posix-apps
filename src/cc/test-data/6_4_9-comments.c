/* clang-format off */
"a//b"
#include "//e"
// */
f = g/**//h;
//\
i();
/\
/ j();
#define glue(x,y) x##y
glue(/,/) k();
/*//*/ l();
m = n//**/o
 + p;
