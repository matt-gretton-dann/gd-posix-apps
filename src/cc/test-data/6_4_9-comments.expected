 
"a 
#include " 
 
f = g /h;
 
 
#define glue(x,y) x##y
glue(/,/) k();
 l();
m = n 
 + p;
